src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -0,0 +1,126 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import PIDControl.PIDControl;
+import PIDControl.PIDControl.Coefficient;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Manipulator extends SubsystemBase 
+{
+    private static Manipulator _instance;
+
+    public static Manipulator getInstance()
+    {
+        if(_instance == null)
+        {
+            _instance = new Manipulator();
+        }
+
+        return _instance;
+    }
+
+    //Wrist Controls
+    private CANSparkMax      _wristMotor;
+    private DutyCycleEncoder _wristEncoder;
+    private PIDControl       _wristPID;
+    //Twist Motion Controls
+    private CANSparkMax      _twistMotor;
+    private DutyCycleEncoder _twistEncoder;
+    private PIDControl       _twistPID;
+    //Hand Controls
+    private CANSparkMax      _graspMotor;
+    private DutyCycleEncoder _graspEncoder;
+    private PIDControl       _graspPID;
+    //Intake Controls
+    private CANSparkMax      _intakeMotor;
+    private DigitalInput     _intakeSensor;
+
+    private Manipulator()
+    {
+        _wristMotor         = new CANSparkMax(Constants.Manipulator.WRIST_MOTOR_CAN_ID, MotorType.kBrushless);
+        _wristEncoder       = new DutyCycleEncoder(Constants.Manipulator.WRIST_ENCODER_PORT);
+        _wristPID           = new PIDControl();
+        _twistMotor         = new CANSparkMax(Constants.Manipulator.TWIST_MOTOR_CAN_ID, MotorType.kBrushless);
+        _twistEncoder       = new DutyCycleEncoder(Constants.Manipulator.TWIST_ENCODER_PORT);
+        _twistPID           = new PIDControl();
+        _graspMotor         = new CANSparkMax(Constants.Manipulator.GRASP_MOTOR_CAN_ID, MotorType.kBrushless);
+        _intakeMotor        = new CANSparkMax(Constants.Manipulator.INTAKE_MOTOR_CAN_ID, MotorType.kBrushless);
+        _intakeSensor       = new DigitalInput(Constants.Manipulator.INTAKE_SENSOR_PORT);
+        _graspEncoder       = new DutyCycleEncoder(Constants.Manipulator.GRASP_ENCODER_PORT);
+        _graspPID           = new PIDControl();
```

Comment:
Add some whitespace in here to group these the same way you do with the variable declarations above the constructor